README.md

```diff
@@ -49,4 +49,4 @@ Contributions, issues, and feature requests are welcome!
 
 ## 📝 License
 
-This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] `required`

- [ ] Kindly  add a `live demo` with the online version of the portfolio .